nuget/AspNetCoreRequestHandler.nuspec

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.AspNetCore.AspNetCoreModule.RequestHandler</id>
+    <title>Microsoft ASP.NET Core Module Request Handler</title>
+    <version>$VERSION$</version>
+    <authors>Microsoft</authors>
+    <owners>Microsoft</owners>
+    <licenseUrl>http://www.microsoft.com/web/webpi/eula/net_library_eula_ENU.htm</licenseUrl>
+    <copyright>© .NET Foundation. All rights reserved.</copyright>
+    <projectUrl>http://www.asp.net/</projectUrl>
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288859</iconUrl>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>ASP.NET Core Module Request Handler</description>
+    <language>en-US</language>
+    <tags>Microsoft.AspNetCore.AspNetCoreModule.RequestHandler</tags>
+  </metadata>
+  <files>
+  <!-- TODO make build output outside of content files. -->
+    <file src="src\RequestHandler\bin\$Configuration$\Win32\aspnetcorerh.dll" target="runtimes\win-x86\native\aspnetcorerh.dll" />
```

Comment:
To make this TFM specific, you can use 'nativeassets'. Example from the SQLite package: https://github.com/natemcmaster/libsqlite3-package/blob/master/SQLite.nuspec#L26-L28

```
runtimes/win-x86/nativeassets/netcoreapp2.1/aspnetcorerh.dll
```